legacy/src/scss/_patterns_button.scss

```diff
@@ -70,6 +70,11 @@ $vanilla-2-icon-button-side-padding: $sph-inner--small * 1.5;
     &:hover {
       background-color: inherit;
       text-decoration: underline;
+
+      // Override inherited button style from Vanilla
+      &:active {
```

Comment:
I think it's okay to move this outside `&:hover` with an `!important` because it's an explicit override. As it is currently you can click on the button, hold the click and move the cursor outside and the button turns grey.

I'm thinking maybe this should make it's way into Vanilla because we're using a lot of links like buttons across the apps. And I'm not a fan of using *actual* links where you have to `event.preventDefault()` and `href="##"` or whatever.

Reply:
Done. I definitely agree about not using links for things that aren't links. I'll propose it to Vanilla for the next working group meeting.